wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -643,11 +643,6 @@
 
 			we.publish(topic.AJAX_CALL_PRECONDITION, attrs);
 
-			if (attrs.mp) { // multipart form. jQuery.ajax() doesn't help here ...
```

Comment:
This one will also change the set of supported browsers: Won't work in IE 9 any more.

Reply:
This is noted in bug description here: https://issues.apache.org/jira/browse/WICKET-6517